collect_app/src/main/res/values/buttons.xml

```diff
@@ -18,6 +18,7 @@
         <item name="android:paddingTop">@dimen/margin_small</item>
         <item name="android:paddingBottom">@dimen/margin_small</item>
         <item name="android:textAppearance">?textAppearanceBodyMedium</item>
+        <item name="iconSize">24dp</item>
```

Comment:
Different icon buttons were using different icon sizes. It wasn’t noticeable before because, even when they appeared on the same screen, they were displayed vertically. In the “add repeat” dialog, however, the buttons are placed in one row, and there the size differences became visible.